src/pkg/packager/components.go

```diff
@@ -251,3 +251,17 @@ func (p *Packager) appendIfNotExists(slice []string, item string) []string {
 	}
 	return append(slice, item)
 }
+
+func (p *Packager) requiresCluster(component types.ZarfComponent) (confirmComponent bool) {
```

Comment:
nit; it's a bit weird to have a named return but not use it.

Reply:
Good find!  Fixed